src/main/webapp/resources/js/apis/remote-api/remote-api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+/**
+ * Check the status of a specific Remote API.
+ * @param {number} id - identifier for the API.
+ * @returns {Promise<boolean>}
+ */
+export function checkConnectionStatus({ id }) {
+  return axios
+    .get(`ajax/remote_api/status/${id}`)
```

Comment:
Use setBaseUrl to set the url  `ajax/remote_api` in case we add to this file in the future

Reply:
Updated in da29063275c8ad8ef8b178984406693579385a5d